Didi/navigationview search (#677)
telerik/maui-docs: controls/navigationview/styling/navigationitem.md

---

> title: NavigationItem Styling

Reviewer: ```suggestion
title: Navigation Item Styling
```

page_title: .NET MAUI NavigationView Documentation - NavigationItem Styling
description: Learn how to style the navigation item inside the .NET MAUI NavigationView Pane.
position: 2
slug: navigationview-item-styling
---

> # .NET MAUI NavigationItem Styling

Reviewer: ```suggestion
# Styling a Navigation Item
```

> Style the navigation item, by setting the `Style` property to the `NavigationViewItem` or use implicit style.

Reviewer: ```suggestion
Style the navigation items by setting the `Style` property to the `NavigationViewItem` or using an implicit style:
```
